### PR TITLE
DOCS-1076: Operator v5.0.11 Release

### LINKED ISSUE #1076: [RELEASE] Operator 5.0.11
**Summary**

A[ clear and concise description of what the problem is. Ex. I'm always frustrated when [...]](https://github.com/minio/operator/releases/tag/v5.0.11)

**Goals**

- [x] Document `MINIO_OPERATOR_IMAGE` envvar override [PR 1854](https://github.com/minio/operator/pull/1854)
- [x] Operator STS enabled by default [PR 1848](https://github.com/minio/operator/pull/1848)
- [x] Additional improvements to TLS behavior in Operator [PR 1847](https://github.com/minio/operator/pull/1847)
- [x] Support for `MINIO_KMS_SECRET_KEY_FILE`, possible spillover to server docs [PR 1789](https://github.com/minio/operator/pull/1789/files)
- [x] `--kes-image` flag for `kubectl minio tenant create` [PR 1808](https://github.com/minio/operator/pull/1808)
- [x] `--retain-namespace` flag for `kubectl minio delete` [PR 1807](https://github.com/minio/operator/pull/1807)

** For Consideration **

- [x] Operator CRD supports PVC deletion on tenant delete - may require some warnings in docs [PR 1828](https://github.com/minio/operator/pull/1828/files)
- [ ] Add guidance for health checks outside of pods post change to UBI 9 [PR 1831](https://github.com/minio/operator/pull/1831)
- [ ]

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #1076 

Smaller than I expected, and a few things I don't think we need to document.

Staged:

- http://192.241.195.202:9000/staging/DOCS-1076/k8s/developers/sts-for-operator.html
- http://192.241.195.202:9000/staging/DOCS-1076/k8s/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.html#kubectl.minio.tenant.create.-kes-image
- http://192.241.195.202:9000/staging/DOCS-1076/k8s/reference/kubectl-minio-plugin/kubectl-minio-tenant-delete.html#kubectl.minio.tenant.delete.-retain-namespace
- http://192.241.195.202:9000/staging/DOCS-1076/k8s/reference/operator-environment-variables.html

### CODE REVIEW

File: source/developers/sts-for-operator.rst [hunk 1]
```diff
@@ -29,7 +29,9 @@ Benefits of STS for MinIO Operator include:
 
 .. important:: 
 
-   MinIO Operator launches with STS *disabled* by default.
+   Starting with Operator v5.0.11, STS is *enabled* by default.
+
+   Previous versions of the Operator start with STS *disabled* by default.
    To use STS with the Operator, you must first explicitly enable it.
```
Comment (djwfyi) [inline source/developers/sts-for-operator.rst:35]:
```suggestion
   To use STS with v5.0.10 or older versions of the Operator, you must first explicitly enable it.
```

**Review by djwfyi — APPROVED** [review]
A few suggestions for clarity for you to consider, otherwise LGTM.

File: source/reference/operator-environment-variables.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+.. envvar:: OPERATOR_STS_ENABLED
+
+   Toggle STS Service ``on`` or ``off``.
+
+   .. versionchanged:: v5.0.11
+
+      When not specified, the default value is ``on``.
+
+   Prior to Operator 5.0.11, the default value is ``off``.
```
[the remark this answers is not in the document]
Comment (ravindk89) [reply source/reference/operator-environment-variables.rst]: I think `was`, though `is` is probably _fine_

File: source/reference/operator-environment-variables.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+.. code-block:: shell
+   :class: copyable
+
+   kubectl set env -n minio-operator deployment/minio-operator <ENV_VARIABLE>=<value> ... <ENV_VARIABLE2>=<value2>
+
+Replace:
+
+- ``minio-operator`` with the namespace for your Operator, if not using the default value.
+- ``deployment/minio-operator`` with the deployment for your Operator, if not the default value.
+  (This is not common.)
```
Comment (feorlen) [inline source/reference/operator-environment-variables.rst:28]: a very weak _maybe_ but an idea:
```suggestion
  (Most deployments use the default value.)
```

**Review by feorlen — APPROVED** [review]
LGTM. One tiny reword you might like

**Pull request merged**